jobs/update_orphaning_dashboard_etl.py

```diff
@@ -373,66 +478,72 @@ def latest_version_on_date(date, major_releases):
-                       "minSubsessionDate": min_subsession_date.strftime("%Y-%m-%d"),
-                       "minSubsessionHours": min_subsession_hours,
-                       "minSubsessionSeconds": min_subsession_seconds,
-                       "minUpdatePingCount": min_update_ping_count}
+report_details_dict = {
+    "latestVersion": latest_version,
+    "upToDateReleases": up_to_date_releases,
+    "minReportDate": min_report_date.strftime("%Y-%m-%d"),
+    "maxReportDate": max_report_date.strftime("%Y-%m-%d"),
+    "weeksOfSubsessionData": weeks_of_subsession_data,
+    "minSubsessionDate": min_subsession_date.strftime("%Y-%m-%d"),
+    "minSubsessionHours": min_subsession_hours,
+    "minSubsessionSeconds": min_subsession_seconds,
+    "minUpdatePingCount": min_update_ping_count,
+}
 report_details_dict
 
 
 # Create the common SQL FROM clause.
 # Note: using the parquet is as fast as using 'FROM longitudinal_vYYYMMDD'
 # and it allows the query to go further back in time.
 
-#longitudinal_from_sql = ("FROM longitudinal_v{} ").format(longitudinal_suffix)
+# longitudinal_from_sql = ("FROM longitudinal_v{} ").format(longitudinal_suffix)
 # longitudinal_from_sql = ("FROM parquet.`s3://telemetry-parquet/longitudinal/v{}` ").format(longitudinal_suffix)
-longitudinal_from_sql = ("FROM longitudinal_shim ")
+longitudinal_from_sql = "FROM longitudinal_shim "
 longitudinal_from_sql
 
 
 # Create the common build.version SQL WHERE clause.
-build_version_where_sql = "(build.version[0] RLIKE '^[0-9]{2,3}\.0[\.0-9]*$' OR build.version[0] = '50.1.0')"
+build_version_where_sql = "(build.version[0] RLIKE '^[0-9]{2,3}\\.0[\\.0-9]*$' OR build.version[0] = '50.1.0')"
 build_version_where_sql
 
 
 # Create the remaining common SQL WHERE clause.
-common_where_sql = (""
+common_where_sql = (
+    ""
     "build.application_name[0] = 'Firefox' AND "
     "DATEDIFF(SUBSTR(subsession_start_date[0], 0, 10), '{}') >= 0 AND "
     "DATEDIFF(SUBSTR(subsession_start_date[0], 0, 10), '{}') < 0 AND "
     "settings.update.channel[0] = '{}'"
-"").format(min_report_date_sql,
-           max_report_date_sql,
-           channel_to_process)
+    ""
+).format(min_report_date_sql, max_report_date_sql, channel_to_process)
 common_where_sql
 
 
 # Create the SQL for the summary query.
-summary_sql = (""
-"SELECT "
-    "COUNT(CASE WHEN build.version[0] >= '{}.' AND build.version[0] < '{}.' THEN 1 END) AS versionUpToDate, "
-    "COUNT(CASE WHEN build.version[0] < '{}.' AND build.version[0] >= '{}.' THEN 1 END) AS versionOutOfDate, "
-    "COUNT(CASE WHEN build.version[0] < '{}.' THEN 1 END) AS versionTooLow, "
-    "COUNT(CASE WHEN build.version[0] > '{}.' THEN 1 END) AS versionTooHigh, "
-    "COUNT(CASE WHEN NOT build.version[0] > '0' THEN 1 END) AS versionMissing "
-"{} "
-"WHERE "
+summary_sql = (
+    ""
+    "SELECT "
+    "COUNT(CASE WHEN build.major_version[0] >= '{}.' AND build.major_version[0] < '{}.' THEN 1 END) AS versionUpToDate, "
```

Comment:
should these still be comparing to `'{}.'` now that there definitely won't be a `.`?

Reply:
and shouldn't these use an integer comparison instead of string?

Reply:
I agree, I think these got automatically casted to integers during my test run.